index.html

```diff
@@ -1,26 +1,81 @@
+				<h1>Relaxr</h1>
+				<nav>
+					<a href="#">About</a>
+					<a href="#">FAQ</a>
+					<a href="#">Team</a>
+					<a href="#">Contact Us</a>
+					<a href="#">Blog</a>
+				</nav>
+			</div>
+		</header>
+		
+		<main class="clearfix">
+			<div class="article">
+				<article>
+					<span class="post">How I implemented Relaxr in 2 weeks and changed my life.</span>
+					
+					<img src="https://github.com/ericmasiello/FEWD31-Assignment3/blob/master/images/blog_photo1.jpg?raw=true">
 
-		<!-- external CSS link -->
-		<link rel="stylesheet" href="css/reset.css">
-		<link rel="stylesheet" href="css/style.css">
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum<span class="more">Read More&gt</span></p>
 
-	</head>
-	<body>
-		<div id="container">
-			<header>
-				<h1>Blank Template</h1>
-				<h2>Just another blank template</h2>
-			</header>
-			<footer>
-				&copy; Footer Goes Here
-			</footer>
-		</div><!-- #container -->
-	</body>
-</html>
\ No newline at end of file
+					<hr>
+
+					<span class="post">I traveled across the globe thanks to Relaxr's Excel automation</span>
+
+					<img src='https://github.com/ericmasiello/FEWD31-Assignment3/blob/master/images/blog_photo2.jpg?raw=true'>
+
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum<span class="more">Read More&gt</span></p>
+					
+					<hr>
+
+				</article>
+
+			</div>
+			
+			<aside class="categories">
+				<h2>Categories</h2>
+				<ul class="list">
+					<li><a href="#">Success Stories</a></li>
+					<li><a href="#">Stats</a></li>
+					<li><a href="#">How tos</a></li>
+					<li><a href="#">Best of</a></li>
+					<li><a href="#">Business tips</a></li>
+				</ul>
+				
+				<h2>About Relaxr</h2>
+
+				<article class="about">Proin sed justo vel sapien varius auctor. Proin scelerisque massa at luctus tincidunt. Fusce porttitor, sapien sed tincidunt fringilla.<span class="learn">Learn More</span></article>
+
+					
+
+					<img src="https://github.com/ericmasiello/FEWD31-Assignment3/blob/master/images/ad-unit.png?raw=true">
+				</aside>
+				</main>
+
+				<div class="sign">
+					<a class="button2" href="#">Sign Up Now!</a>
+				</div>
+
+				<footer>
+					<h4>Relaxr</h4>
```

Comment:
`h4` tags (along with all of `h#` tags) by default have top margin on them. This is what was causing the white space between your `footer` and your `<div class="sign">`. You can remove it with the following

```css
h4 {
 margin-top: 0;
}
```

or more broadly just do this:

```css
h1, h2, h3, h4, h5, h6 {
 margin-top: 0;
}
```